IOS/SwiftSeedProject/SwiftSeedProject/Application/Service/SourceService.swift

```diff
@@ -7,10 +7,31 @@
 //
 
 import Foundation
+import SwiftyJSON
 
 final class SourceService: BaseService<Source> {
     
     public override func updateLocalStoreWithServerInfo() {
-        //TODO: We should add the API CALL HERE
+        api.getSources(language: Language.English) { [weak self] result in
+            do {
+                guard let strongSelf = self else {
+                    return
+                }
+                let json = try result.unwrap()
+                let sources = json["sources"].arrayValue
+                sources.forEach { json in
+                    let id = json["id"].stringValue
+                    if let source = strongSelf.getEntityBy(identifier: id) as Source? {
+                        source.updateWithJSON(json: json)
```

Comment:
I agree with marce comment, maybe we should create another class to delegate some logic